assets/xml/archives/icon_item_static.xml

```diff
@@ -1,194 +1,221 @@
+        <Texture Name="gBowIceIconTex" OutName="bow_ice_icon" Format="rgba32" Width="32" Height="32" Offset="0x4B000" />
+        <Texture Name="gBowLightIconTex" OutName="bow_light_icon" Format="rgba32" Width="32" Height="32" Offset="0x4C000" />
+        <Texture Name="gKokiriSwordIconTex" OutName="kokiri_sword_icon" Format="rgba32" Width="32" Height="32" Offset="0x4D000" />
+        <Texture Name="gRazorSwordIconTex" OutName="razor_sword_icon" Format="rgba32" Width="32" Height="32" Offset="0x4E000" />
+        <Texture Name="gGildedSwordIconTex" OutName="gilded_sword_icon" Format="rgba32" Width="32" Height="32" Offset="0x4F000" />
+        <Texture Name="gFierceDeitySwordIconTex" OutName="fierce_deity_sword_icon" Format="rgba32" Width="32" Height="32" Offset="0x50000" />
+        <Texture Name="gHerosShieldIconTex" OutName="heros_shield_icon" Format="rgba32" Width="32" Height="32" Offset="0x51000" />
+        <Texture Name="gMirrorShieldIconTex" OutName="mirror_shield_icon" Format="rgba32" Width="32" Height="32" Offset="0x52000" />
+        <Texture Name="gQuiver30IconTex" OutName="quiver_30_icon" Format="rgba32" Width="32" Height="32" Offset="0x53000" />
+        <Texture Name="gQuiver40IconTex" OutName="quiver_40_icon" Format="rgba32" Width="32" Height="32" Offset="0x54000" />
+        <Texture Name="gQuiver50IconTex" OutName="quiver_50_icon" Format="rgba32" Width="32" Height="32" Offset="0x55000" />
+        <Texture Name="gBombBag20IconTex" OutName="bomb_bag_20_icon" Format="rgba32" Width="32" Height="32" Offset="0x56000" />
+        <Texture Name="gBombBag30IconTex" OutName="bomb_bag_30_icon" Format="rgba32" Width="32" Height="32" Offset="0x57000" />
+        <Texture Name="gBombBag40IconTex" OutName="bomb_bag_40_icon" Format="rgba32" Width="32" Height="32" Offset="0x58000" />
+        <Texture Name="gDefaultWalletIconTex" OutName="default_wallet_icon" Format="rgba32" Width="32" Height="32" Offset="0x59000" />
+        <Texture Name="gAdultsWalletIconTex" OutName="adults_wallet_icon" Format="rgba32" Width="32" Height="32" Offset="0x5A000" />
+        <Texture Name="gGiantsWalletIconTex" OutName="giants_wallet_icon" Format="rgba32" Width="32" Height="32" Offset="0x5B000" />
+        <Texture Name="gFishingRodIconTex" OutName="fishing_rod_icon" Format="rgba32" Width="32" Height="32" Offset="0x5C000" />
+        <Texture Name="gOdolwaRemainsIconTex" OutName="odolwa_remains_icon" Format="rgba32" Width="32" Height="32" Offset="0x5D000" />
+        <Texture Name="gGohtRemainsIconTex" OutName="goht_remains_icon" Format="rgba32" Width="32" Height="32" Offset="0x5E000" />
+        <Texture Name="gGyorgRemainsIconTex" OutName="gyorg_remains_icon" Format="rgba32" Width="32" Height="32" Offset="0x5F000" />
+        <Texture Name="gTwinmoldRemainsIconTex" OutName="twinmold_remains_icon" Format="rgba32" Width="32" Height="32" Offset="0x60000" />
```

Comment:
Similar to my comment above about the Moon's Tear, these are called "Odolwa's Remains", "Goht's Remains", etc. in-game. I can see why you might just skip that if you don't want to mess with the apostrophes though.

Reply:
I think it's better to have the apostrophe actually. I'm going through `item_name_static` right now where it actually has the text of the names in a texture and it wouldn't be right if there were letters omitted from that. And might as well be consistent between `item_name_static` and `icon_item_static`

Reply:
I think what we have been doing is keeping the `s`, but not actually including the apostrophe. For example, object_gi_reserve00 uses `MoonsTear` as the prefix.

Reply:
Yeah, that's what I kind of expected too. I'm not sure if putting the apostrophe in would cause problems when used as a symbol, but we can at least keep the "s".